src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
+const body = document.querySelector('body');
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
```

Comment:
```suggestion
  const notification = document.createElement('div');
```
use better variable names instead of tag names

Reply:
check all such cases